src/bin/pg_autoctl/pgsql.c

```diff
@@ -1310,6 +1320,41 @@ pgsql_get_wal_lag_from_standby(PGSQL *pgsql, int64_t *walLag)
 }
 
 
+/*
+ * pgsql_get_received_lsn_from_standby queries a standby PostgreSQL server to get the
+ * received_lsn value from the pg_stat_wal_receiver system view.
+ *
+ * received_lsn is the latest lsn known to be received and flushed to the disk. It does
+ * not specify if it is applied or not. Caller should have allocated necessary memory
+ * for result value.
+ */
+bool
+pgsql_get_received_lsn_from_standby(PGSQL *pgsql, char *received_lsn)
```

Comment:
When passing a `char *` to be filled-in, all the other call sites would add the length of the passed-in buffer as a separate argument, usually named `size`. Please do the same thing here, otherwise the caller must know that this argument is hard-coded to take NAMEDATALEN bytes.

Reply:
ok